client/js/restaurants_api.js

```diff
@@ -1043,6 +1166,55 @@ function create_reservations_table(reservationsResponse)
 
 
 
+function create_organization_reservations_table(reservationsResponse)
+{
+	reservationsResponse= JSON.parse(reservationsResponse);
+
+	//clean previous table content
+	var myNode = document.getElementById("reservations_table_body");
+		while (myNode.firstChild) {
+		    myNode.removeChild(myNode.lastChild);
```

Comment:
this could be really, really slow! please avoid that

use table_body.innerHTML = ''